utils/feed_fetcher.py

```diff
@@ -34,7 +34,7 @@
 from django.utils.encoding import smart_unicode
 from utils import json_functions as json
 from celery.exceptions import SoftTimeLimitExceeded
-from vendor import tweepy
```

Comment:
Can you undo these two? Tweepy is explicitly vendorized because of some changes that I needed to make.